packages/vats/test/upgrading/test-upgrade-vats.js

```diff
@@ -0,0 +1,56 @@
+import { test as anyTest } from '@agoric/swingset-vat/tools/prepare-test-env-ava.js';
+import { buildVatController } from '@agoric/swingset-vat';
+
+/** @type {import('ava').TestFn<Awaited<ReturnType<typeof makeTestContext>>>} */
+const test = anyTest;
+
+/**
+ * WARNING: uses ambient authority
+ * of import.meta.url
+ */
+const makeTestContext = async () => {
+  const bfile = name => new URL(name, import.meta.url).pathname;
+  return { bfile };
```

Comment:
consider `bundlePath`

Reply:
`bfile` followed precedent from a bunch of tests that I looked at. But if that precedent wasn't already in your mind, I suppose it's not sufficiently relevant.